### PR TITLE
Show date range in history

### PR DESCRIPTION
**Pull request by cb372**
* Add the date range info to the JobHistory and RunningJob models and Dynamo tables
* Display that info in the job history table

screen shot 2016-04-18 at 16 48 21

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
We optimistically add a job history when we add a reindex here: https://github.com/guardian/floodgate/blob/master/public/components/reindexController.react.js#L97

Not sure if it matters in your case(?) but thought it was worth highlighting.

File: public/components/jobHistoryDateRange.react.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    nonEmpty(value) {
+        return !this.empty(value);
+    }
+
+    render () {
+        if (this.empty(this.props.rangeFrom) && this.empty(this.props.rangeTo)) {
+            return <span>All time</span>;
+        } else if (this.nonEmpty(this.props.rangeFrom) && this.empty(this.props.rangeTo)) {
+            return <span>{ new Date(this.props.rangeFrom).toISOString().slice(0,10) + " - now" }</span>;
```
Comment (LATaylor-guardian) [inline public/components/jobHistoryDateRange.react.js]: This has no benefit but you might find moment.js preferable to use. See: https://github.com/guardian/floodgate/blob/ef2330020d3ab640101b580483c2493adc5b40f4/public/components/reindexForm.react.js#L12

**Comment by LATaylor-guardian** [comment]
Looks good! Useful information. Aside from my query 👍

File: public/components/jobHistoryDateRange.react.js (hunk 1 above)
Comment (cb372) [inline public/components/jobHistoryDateRange.react.js]: Thanks, that's much nicer.

**Comment by cb372** [comment]
>We optimistically add a job history

Good point. I've added code to handle that case as well.

**Pull request merged**